src/traces/sankey/plot.js

```diff
@@ -143,6 +144,7 @@ module.exports = function plot(gd, calcData) {
     var outgoingLabel = _(gd, 'outgoing flow count:') + ' ';
 
     var linkHoverFollow = function(element, d) {
+        if(gd._fullLayout.hovermode === false) return;
```

Comment:
Do we need to bail out of the `unhover` handlers as well, to ensure that we don't emit `plotly_unhover` events? Or does this already avoid these events?

Reply:
@alexcjohnson It was indeed emitting a bunch of `plotly_unhover` events. It is fixed in my next commit.

Reply:
Great! I guess then we should 🔒 the events too, by adding a `hovermode: false` case to the [click/hover events test](https://github.com/plotly/plotly.js/blob/45f06c447af2bc6b5e7196b2b949d6b4b4737ca6/test/jasmine/tests/sankey_test.js#L509)